src/main/kotlin/no/nav/familie/ef/iverksett/arena/JmsConfig.kt

```diff
@@ -0,0 +1,74 @@
+package no.nav.familie.ef.iverksett.arena
+
+import com.ibm.mq.constants.CMQC
+import com.ibm.mq.jms.MQConnectionFactory
+import com.ibm.mq.jms.MQQueueConnectionFactory
+import com.ibm.msg.client.wmq.WMQConstants
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.jms.annotation.EnableJms
+import org.springframework.jms.connection.CachingConnectionFactory
+import org.springframework.jms.connection.UserCredentialsConnectionFactoryAdapter
+import org.springframework.jms.core.JmsTemplate
+import javax.jms.ConnectionFactory
+import javax.jms.JMSException
+
+@Configuration
+@EnableJms
+class JmsConfig {
+
+    private val UTF_8_WITH_PUA = 1208
+
+    @Value("\${MQ_QUEUE_MANAGER}")
+    lateinit var queueManager: String
```

Comment:
Her tenker jeg att det går å bruke
```
@EnableJms
@ConfigurationProperties("mq")
@ConstructorBinding
class MQConfig(private val queueManager: String,
                          private val hostname: String) {
```

Då får man en lite ryddigere application.yml med

```
mq:
  queueManager: ...
  hostname: ...
  username: ${SERVICEBRUKER}
```

Reply:
Kanskje fint å kalle det `arena-mq` også - så blir det eksplisitt hvem denne konfigurasjonen er ment for?

Reply:
Smart! I fix.